lib/membrane_webrtc/ex_webrtc/sink.ex

```diff
@@ -133,6 +133,38 @@ defmodule Membrane.WebRTC.ExWebRTCSink do
     raise "Track #{inspect(track_id)} was rejected by the other peer"
   end
 
+  @impl true
+  def handle_info({:ex_webrtc, _from, {:rtcp, rtcp_packets}}, ctx, state) do
+    request_keyframe? =
+      rtcp_packets
+      |> Enum.reduce(false, fn packet, request_keyframe? ->
+        case packet do
+          {_track_id, %ExRTCP.Packet.PayloadFeedback.PLI{}} ->
+            Membrane.Logger.debug("Keyframe request received: #{inspect(packet)}")
+            true
+
+          _other ->
+            Membrane.Logger.debug("Ignoring RTCP packet: #{inspect(packet)}")
+            request_keyframe?
+        end
+      end)
+
+    actions =
+      if request_keyframe? do
+        ctx.pads
+        |> Enum.flat_map(fn {pad_ref, pad_data} ->
+          case pad_data.options.kind do
+            :video -> [event: {pad_ref, %Membrane.KeyframeRequestEvent{}}]
```

Comment:
Shouldn't we somehow 'debounce' the keyframe requests? AFAIR the browser is going to spam us with PLIs when it's unable to decode the stream to make sure we receive the PLI ASAP. And we shouldn't generate as many keyframes as PLIs. cc @mickel8

Reply:
Sounds resonable but how to distinct between PLI retransmission and real PLI :thinking: